Server/server.js

```diff
@@ -19,6 +25,7 @@ app.get('/api/funding_round', function(req, res) {
 	})
 });
```

Comment:
As previously mentioned,  it may be good to have the find company logic in the Funding.js module.

Reply:
Also, I think deleting line 19 and reducing indentation for lines 20-25 could make the code look cleaner.